app/serializers/books/index_serializer.rb

```diff
@@ -0,0 +1,13 @@
+module Books
+  class IndexSerializer < ActiveModel::Serializer
+    attributes :id, :author, :title, :image, :publisher, :year, :genre, :created_at, :updated_at
+
+    def image
```

Comment:
if the image is already the name of the field  that has the URL why would you make ""harder"" to reach the image, because  because maybe  the front-end team will have to do `books[0].image.url` when they can have it in `books[0].image`